test/test_cli.rb

```diff
@@ -25,6 +25,8 @@ def setup
 
     @events = Puma::Events.new
     @events.on_booted { @ready << "!" }
+
+    @puma_vers_re = "\\d+.\\d+.\\d+(\\.[a-z\\d]+)?"
```

Comment:
```suggestion
    @puma_version_pattern = "\\d+.\\d+.\\d+(\\.[a-z\\d]+)?"
```

Reply:
Fixed, pushed back into `test_cli.rb - allow for Puma version like 6.0.1.dev or 6.0.1.rc1`